### ISSUE TITLE
How to calculate the beat of dance ~

### ISSUE THREAD

**Issue by WilliammmZ** [stonyhu/DanceRevolution#14]
Thx for ur work. It is very hard to follow if there no evaluation code ~

**Comment by stonyhu** [comment]
@WilliammmZ Thank you for your interest in our work. For beat coverage and beat hit rate, we follow [Yalta et al., 2019](https://github.com/Fhrozen/motion_dance) to calculate and you can refer to its implementation. We are archiving the evaluation code and will share recently. But according to our experience, the current evaluation method is not good enough in practice. This could be the future research work to the community.

**Comment by stonyhu** [comment]
@WilliammmZ The evaluation code is released, you can use the style classifier trained by our code to extract the feature vector to calculate automatic metrics.

**Comment by GuHuangAI** [comment]
> @WilliammmZ The evaluation code is released, you can use the style classifier trained by our code to extract the feature vector to calculate automatic metrics.

Excuse me, could you please provide the pre-trained weight of the style classifier?

**Comment by stonyhu** [comment]
The original models, including dance generation model and classification model, are missed due to my previous team change. The released generation model is the one that I recently reproduced. You can simply train the classifier by the provided codes.

**Comment by stonyhu** [comment]
The original models including dance generation model and classification model are missed on the GPU server. But you can simply train the classifier by the provided code in the repo.
